app/views/custom_fields/edit.html.erb

```diff
@@ -33,9 +33,13 @@ See COPYRIGHT and LICENSE files for more details.
 
 <%= error_messages_for 'custom_field' %>
 
-<%= labelled_tabular_form_for @custom_field, as: :custom_field,
-                              url: custom_field_path(@custom_field),
-                              html: {method: :put, id: 'custom_field_form'} do |f| %>
-  <%= render partial: 'form', locals: { f: f } %>
-  <%= styled_button_tag t(:button_save), class: '-primary -with-icon icon-checkmark' %>
+<% if @custom_field.field_format == "hierarchy" %>
```

Comment:
🟢 It would be nice to have encapsulated predicates: string vs symbol comparisons are easy pitfals.

```ruby
<% if @custom_field.field_format_hierarchy? %>
```

Reply:
good idea, I will introduce that.